gst/droidcamsrc/gstdroidcamsrc.c

```diff
@@ -2207,3 +2226,53 @@ gst_droidcamsrc_find_picture_resolution (GstDroidCamSrc * src,
 
   return ret;
 }
+
+gboolean
+gst_droidcamsrc_is_zsl_enabled (GstDroidCamSrc * src)
+{
+  return src->image_mode & GST_DROIDCAMSRC_IMAGE_MODE_ZSL
+      || src->image_mode & GST_DROIDCAMSRC_IMAGE_MODE_ZSL_AND_HDR;
+}
+
+gboolean
+gst_droidcamsrc_is_hdr_enabled (GstDroidCamSrc * src)
+{
+  return src->image_mode & GST_DROIDCAMSRC_IMAGE_MODE_HDR
+      || src->image_mode & GST_DROIDCAMSRC_IMAGE_MODE_ZSL_AND_HDR;
+}
+
+
+static gboolean
+gst_droidcamsrc_is_zsl_and_hdr_supported (GstDroidCamSrc * src)
```

Comment:
I feel that these should really be something like:
gst_droidcamsr_supported_modes() or something that would return array what is supported.

And similar thing for above gst_droidcamsr_enabled_modes() that would return what is enabled.

As if there will be more modes available in the future these will not really scale if you need to do separate functions for:
a_and_b
b_and_c
a_and_c
a_and_b_and_c
...